src/main/java/dansplugins/factionsystem/eventhandlers/DamageHandler.java

```diff
@@ -170,6 +212,10 @@ private boolean wasProjectileShotByPlayer(EntityDamageByEntityEvent event) {
         return projectileSource instanceof Player;
     }
 
+    private boolean wasDamageBetweenEntities(EntityDamageByEntityEvent event) {
```

Comment:
Since we're checking if the damager was a player, the name of this method should ideally reflect that.

Reply:
done